tests/stockfish/test_models.py

```diff
@@ -277,7 +292,9 @@ def test_get_fen_position_after_some_moves(self, stockfish):
         )
 
     def test_get_stockfish_major_version(self, stockfish):
-        assert stockfish.get_stockfish_major_version() in (8, 9, 10, 11, 12, 13, 14)
+        assert (
+            stockfish.get_stockfish_major_version() in (8, 9, 10, 11, 12, 13, 14)
+        ) != stockfish.is_stockfish_major_version_a_date()
```

Comment:
```suggestion
        assert stockfish.get_stockfish_major_version() in (8, 9, 10, 11, 12, 13, 14)
        assert stockfish.is_development_build_of_engine() is False
```

Reply:
The idea here was to have the test still be able to pass if the person running pytest was using a development build of SF. In this case the major version wouldn't be in the range of [8, 14], but it would be a date.

Reply:
👌🏻